src/Kros.AspNetCore/Extensions/ConfigurationBuilderExtenstions.cs

```diff
@@ -30,47 +30,46 @@ public static IConfigurationBuilder AddLocalConfiguration(this IConfigurationBui
         /// <param name="refreshConfiguration">A callback used to configure Azure App Configuration refresh options.</param>
         /// <returns>The same instance of the Microsoft.Extensions.Hosting.IHostBuilder for chaining.</returns>
         /// <remarks>
-        /// Configuration should contain attributes AppConfig:Endpoint and AppConfig:Settings.
+        /// Configuration should contain <b>AppConfig</b> section, which is mapped to <see cref="AppConfigOptions "/> class.
         /// </remarks>
         public static IConfigurationBuilder AddAzureAppConfig(
             this IConfigurationBuilder config,
             string environmentName,
             Action<AzureAppConfigurationRefreshOptions> refreshConfiguration = null)
         {
-            var settings = config.Build();
+            IConfigurationRoot settings = config.Build();
+            AppConfigOptions appConfig = new();
+            settings.Bind("AppConfig", appConfig);
 
-            string appConfigEndpoint = settings["AppConfig:Endpoint"];
-            if (string.IsNullOrWhiteSpace(appConfigEndpoint))
+            if (string.IsNullOrWhiteSpace(appConfig.Endpoint))
             {
                 return config;
             }
 
             config.AddAzureAppConfiguration(options =>
             {
-                var credential = new DefaultAzureCredential();
+                DefaultAzureCredentialOptions credentialOptions = new()
+                {
+                    ManagedIdentityClientId = appConfig.IdentityClientId
```

Comment:
Keď `"identityClientId"` nie je vyplnené, je to OK?

Reply:
Hej, to je Ok. AK ho nezadáš, tak štandardne nie je vyplnené. Ale pre istotu si to ešte raz testnem, pretože myslím že inak som tam mával `null` a nie prázdny reťazec. Ale verím, že nie sú taký blbý, aby v tom robili rozdiel. :)